### PR TITLE
Feature/adapt for ghes endpoint

### PR DESCRIPTION
**Pull request by qminhpho**

### CODE REVIEW

File: main.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ async function run(): Promise<void> {
     const team = core.getInput('team') || process.env.team || ''
     let repo = core.getInput('repo') || process.env.repo || ''
     let owner = core.getInput('owner') || process.env.owner || ''
+    let BASE_URL = process.env.GITHUB_API_URL || 'https://api.github.com'
```
Comment (rajbos) [inline main.ts:16]: Can you add a comment in here to tell WHY we need to do this? It's not clear now, unless you know about it.

File: README.md [hunk 2]
```diff
@@ -2,17 +2,27 @@
...
 
 Sometimes you want to tag a team or multiple persons when an issue (or something else) is created. This action will help you do that by tagging them in an issue.
+##### Note: only tagging a single person or team is currently supported.
 
 
 # Usage:
 
+Requirements:
+
+A GitHub Apps [https://docs.github.com/en/developers/apps/getting-started-with-apps/about-apps#about-github-apps] installed on the repository/organization that the GitHub Actions Workflow will execute from
```
Comment (rajbos) [inline README.md:12]:
```suggestion
A GitHub App [https://docs.github.com/en/developers/apps/getting-started-with-apps/about-apps#about-github-apps] installed on the repository/organization that the GitHub Actions Workflow will execute from
```

Let's fix that typo: it should be singular

**Review by rajbos — COMMENTED** [review]
Thanks for the contribution! Just some minor text fixes and then we are good to go!

File: main.ts (hunk 1 above)
Comment (qminhpho) [reply main.ts:16]: agree, will do

**Comment by rajbos** [comment]
@qminhpho: Do note: we still miss the main.js conversion in here, but I can add it if needed :-D

File: dist/main.js [hunk 3]
```diff
@@ -14667,7 +14667,7 @@ function run() {
       const team = core.getInput("team") || process.env.team || "";
       let repo = core.getInput("repo") || process.env.repo || "";
       let owner = core.getInput("owner") || process.env.owner || "";
-      let BASE_URL = process.env.GITHUB_API_URL || "https://api.github.com";
```
Comment (rajbos) [inline dist/main.js]: I dont like this removal: without we cannot test it locally out of the box :-(. Can you add it again?

**Review by rajbos — APPROVED** [review]
Awesome!

**Pull request merged**